src/Hedgehog/Range.fs

```diff
@@ -174,9 +176,9 @@ module Range =
     let inline exponentialFrom (z : 'a) (x : 'a) (y : 'a) : Range<'a> =
         Range (z, fun sz ->
             let x_sized =
-                clamp x y (scaleExponential sz z x)
+                scaleExponential x y sz z x
             let y_sized =
-                clamp x y (scaleExponential sz z y)
+                scaleExponential x y sz z y
```

Comment:
If we can avoid passing `x` twice, we should be able to take this as-is for now.

Reply:
Since one call has only x repeated as the last parameter, and the other has only y repeated, I'm not sure how that could be done (other than partially applying the function)